src/lib.rs

```diff
@@ -10,25 +10,39 @@ mod ast;
 mod avro;
 mod bigquery;
 mod jsonschema;
+mod traits;
 
 use serde_json::{json, Value};
+use traits::TranslateFrom;
 
-fn into_ast(input: &Value) -> ast::Tag {
+#[derive(Copy, Clone)]
+pub enum ResolveMethod {
+    Cast,
+    Drop,
+    Panic,
+}
+
+#[derive(Copy, Clone)]
+pub struct Context {
```

Comment:
Are there plans or ideas where the context might contain more than just this single field?
While we're at it, why not add a quick doc comment explaining what a context is and what the field configures.

Reply:
Another example would be #74. In essence, the context is used to pass flags from the command-line into the logic.